GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -0,0 +1,3472 @@
+            2.70982187469671100e-02_dp, 5.98347883444798334e-02_dp, 1.26391629758851071e-01_dp, 2.57116152461817626e-01_dp,&
+            5.06388230347464385e-01_dp, 9.70039638261550818e-01_dp, 1.81673675871043816e+00_dp, 3.35684316451570375e+00_dp,&
+            6.28650545297204477e+00_dp, 1.24303076540154611e-05_dp, 6.12924753893748605e-05_dp, 2.22338565270176626e-04_dp,&
+            6.73725996984385692e-04_dp, 1.80782224834487824e-03_dp, 4.44029321814444489e-03_dp, 1.01885563508556497e-02_dp,&
+            2.21389857406311917e-02_dp, 4.59949144505170382e-02_dp, 9.20126454807105032e-02_dp, 1.78222523273414285e-01_dp,&
+            3.35808992061634781e-01_dp, 6.18631233760847832e-01_dp, 1.12372183807606851e+00_dp, 2.05773586398375707e+00_dp,&
+            4.14100882646687207e+00_dp, 3.70975867485438758e-06_dp, 3.23095974345838857e-05_dp, 1.50751791796946199e-04_dp,&
+            5.40915866129935237e-04_dp, 1.65069566621838817e-03_dp, 4.49801466629874431e-03_dp, 1.12590458181901151e-02_dp,&
+            2.63650642132821425e-02_dp, 5.84859131899247639e-02_dp, 1.24024822968737125e-01_dp, 2.53145535590468929e-01_dp,&
+            5.00016351298351092e-01_dp, 9.60271964584979987e-01_dp, 1.80247354207944044e+00_dp, 3.33705426048128562e+00_dp,&
+            6.26035148960221832e+00_dp, 1.08336000906553486e-05_dp, 5.61646913829607426e-05_dp, 2.08706414643022598e-04_dp,&
+            6.41498436540636542e-04_dp, 1.73842323177099346e-03_dp, 4.30149368219168854e-03_dp, 9.92748802913993866e-03_dp,&
+            2.16736122289964903e-02_dp, 4.52053172738387798e-02_dp, 9.07355616494632117e-02_dp, 1.76256045479438767e-01_dp,&
+            3.32938764907261542e-01_dp, 6.14690683320622044e-01_dp, 1.11868286357054081e+00_dp, 2.05176383786785221e+00_dp,&
+            4.13421515627375502e+00_dp, 3.29307433796630773e-06_dp, 2.97376527986368188e-05_dp, 1.41976719845007126e-04_dp,&
+            5.16576765783947141e-04_dp, 1.59126867541210652e-03_dp, 4.36579262671051883e-03_dp, 1.09856221527627942e-02_dp,&
+            2.58330132577597980e-02_dp, 5.75039035275383276e-02_dp, 1.22297087801244098e-01_dp, 2.50240360996554667e-01_dp,&
+            4.95345024579851689e-01_dp, 9.53099040349520976e-01_dp, 1.79198460975399865e+00_dp, 3.32248583508375628e+00_dp,&
+            6.24108223942480667e+00_dp, 9.73921315654793179e-06_dp, 5.25819992873462335e-05_dp, 1.99028066423316980e-04_dp,&
+            6.18407632563333165e-04_dp, 1.68838246973765458e-03_dp, 4.20092557580681921e-03_dp, 9.73759613841949229e-03_dp,&
+            2.13340343901398007e-02_dp, 4.46276041650120678e-02_dp, 8.97990158818834650e-02_dp, 1.74811044817542116e-01_dp,&
+            3.30826015322738642e-01_dp, 6.11785841611682035e-01_dp, 1.11496412843897841e+00_dp, 2.04735361117476744e+00_dp,&
+            4.12919801960688915e+00_dp, 2.98964795412928161e-06_dp, 2.78510556032888645e-05_dp, 1.35463612561978952e-04_dp,&
```

Comment:
I ran a couple of tests with this branch and my GWR code and the results are OK although I haven't covered all the cases.
My main concern is that in this new version we have increased a lot the number of continuation lines and gfortran prints several warnings such as:


```sh
Warning: Limit of 255 continuations exceeded in statement at (1)
/Users/gmatteo/git_repos/greenX/GX-TimeFrequency/src/minimax_tau.F90:2812:1:
```

Note that even in F2003, a maximum of 255 continuation lines is allowed , see e..g. [fortranwiki](https://fortranwiki.org/fortran/show/Continuation+lines)

The fact that gcc and ifort allows for more continuation lines does not mean that other compilers will accept it by default. 
For instance, [NAG](https://www.nag.com/nagware/np/r70_doc/manual/compiler_5_2.html)  requires  `-maxcontin=N`.

I believe we should adhere to the F2003 specs in order to avoid possible portability issues

Reply:
Thanks for the comment. I do not think this warning is problematic. My main points are:

- These are arrays of constants, they are processed at compile time, so there is no impact in performance.
- The maximum number of continuation lines and the maximum length of the line are effectively legacy Fortran features. In fact, there is an [accepted proposal](https://j3-fortran.org/doc/year/19/19-138r1.txt) for improvement of Fortran that removes the 132 and 255 limits. This will be in the next version of the standard. Quote literal:

`
{ 6.3.2.1 Free form line length }

[49:13-14p1] replace the sentence
"If a line consists entirely of characters of default kind (7.4.4),
it shall contain at most 132 characters."

with:
"A line shall contain at most ten thousand characters."
`
and:
`
{ 6.3.2.6 Free form statements }

[50:30p2] replace the sentence
"A statement shall not have more than 255 continuation lines."

with the sentence
"A statement shall not have more than one million characters."

{ new statement length rule }
`